### PR TITLE
Multi tenancy heroku

### PR DESCRIPTION
**Pull request by jonatasrancan**

### CODE REVIEW

**Comment by jonatasrancan** [comment]
@nakedsushi This PR, only the commits after the `Drop Procfile in` are the changes related to heroku

File: config/store_manage.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-  ffmpeg_path: '/usr/local/bin/ffmpeg'
+  ffmpeg_path: '/app/vendor/ffmpeg/ffmpeg'
 
 development:
   ffmpeg_path: '/usr/local/bin/ffmpeg'
 
 test:
-  ffmpeg_path: '/usr/local/bin/ffmpeg'
\ No newline at end of file
+  ffmpeg_path: '/usr/local/bin/ffmpeg'
```
Comment (jonatasrancan) [inline config/store_manage.yml]: I think we could move this paths to `ENV` variables

File: config/initializers/resque.rb [hunk 2]
```diff
@@ -1 +1 @@
-Resque.inline = ENV['RAILS_ENV'] == "development"
+Resque.inline = ENV['RAILS_ENV'] == "development" || ENV['JOB_SYNC']
```
Comment (jonatasrancan) [inline config/initializers/resque.rb]: We added this, based on our conversation yesterday, to run the job synchronous or not.

File: config/store_manage.yml (hunk 1 above)
Comment (nakedsushi) [reply config/store_manage.yml]: Agreed

**Comment by andreibondarev** [comment]
I think you can upmerge this one with master

**Comment by nakedsushi** [comment]
Merged manually: https://github.com/ProjectResound/resound-api/commit/19e270f10657e57091a5575982f49c16ffc5ea58

**Pull request closed without merging**